nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/processor/SimpleProcessLogger.java

```diff
@@ -448,4 +455,13 @@ public void log(LogLevel level, String msg, Object[] os, Throwable t) {
         }
     }
 
+    private String getCauses(final Throwable throwable) {
+        final LinkedList<String> causes = new LinkedList<>();
+        for (Throwable t = throwable; t != null; t = t.getCause()) {
+            causes.push(t.toString());
+        }
+        Collections.reverse(causes);
```

Comment:
I think this was my suggestion so my bad.
`causes.push(t.toString());` already establishes a reversed order because `push` adds the element to the _front_.
So `Collections.reverse(causes);` is unnecessary (it actually reverses _back_ the order of elements).